.env.example

```
@@ -1,3 +1,13 @@
 DATABASE_URI=your-connection-string-here
 PAYLOAD_SECRET=YOUR_SECRET_HERE
+
+# Mongo Express Configuration
+ME_CONFIG_MONGODB_SERVER=your-mongo-server
+ME_CONFIG_MONGODB_PORT=your-mongo-port
+ME_CONFIG_BASICAUTH_USERNAME=your-username
+ME_CONFIG_BASICAUTH_PASSWORD=your-password
+
+# Stripe Configuration
+STRIPE_SECRET_KEY="your-stripe-secret-key-here"
+STRIPE_WEBHOOK_WECRET="your-stripe-webhook-secret-here"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)